_playwright-tests/UI/PopularRepo.spec.ts

```diff
@@ -22,52 +22,29 @@ test.describe('Popular Repositories', () => {
       await expect(page.getByRole('button', { name: 'Add selected repositories' })).toBeVisible();
     });
 
-    await test.step('Select EPEL 10', async () => {
-      await page
-        .getByRole('row', { name: repoName10 })
-        .getByLabel('Select row 0', { exact: true })
-        .click();
-      await page.getByTestId('add-selected-dropdown-toggle-no-snap');
-      await page.getByRole('menuitem', { name: 'Add 1 repositories without snapshotting' });
-    });
-
-    await test.step('Select EPEL 9', async () => {
-      await page
-        .getByRole('row', { name: repoName9 })
-        .getByLabel('Select row 1', { exact: true })
-        .click();
-      await page.getByTestId('add-selected-dropdown-toggle-no-snap');
-      await page.getByRole('menuitem', { name: 'Add 2 repositories without snapshotting' });
-    });
-
-    await test.step('Select EPEL 8 and add both repos', async () => {
-      await page
-        .getByRole('row', { name: repoName8 })
-        .getByLabel('Select row 2', { exact: true })
-        .click();
-      await page.getByTestId('add-selected-dropdown-toggle-no-snap').click();
-      await page.getByRole('menuitem', { name: 'Add 3 repositories without snapshotting' }).click();
-    });
+    // Select the all 3 popular repositories and add them without snapshotting.
```

Comment:
this block (lines 26-36) can be included in its own test.step, maybe something like `await test.step('Add the 3 popular repos without snapshotting', async () => {...});`. then you can remove the `await test.step(Select ${repos[i]}` inside the for loop, and probably the comment at line 25 too